src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from "react"
+import styles from "./Breadcrumb.module.css"
+
+interface Props {
+  children: ReactNode
```

Comment:
I'm wondering if it makes sense for this component to accept children since we are configuring links with an aray (on "BreadcrumbItem"). If we stick with that form of configuration, I would suggest passing it here instead of to a child component (I don't think we'd need the child componet—exposed at least).

That said, if we want to switch to the composable style that we've used for Tabs, then we'll at least want to constrain children to only accept `BreadcrumbItem`s.